data-raw/process_data_sets.R

```diff
@@ -272,3 +272,27 @@ pennies_resamples <-
   select(replicate, everything()) %>%
   unnest(cols = c(data))
 usethis::use_data(pennies_resamples, overwrite = TRUE)
+
+# Electric vehicle charging sessions
+# information from 3,395 high resolution electric vehicle charging sessions
+# Original data from: https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/NFPQLW
+ev_charging <-
+  read_csv("data-raw/station_data_dataverse.csv") %>%
+  mutate(
+    facilityType = case_when(
```

Comment:
I think you can do this in one fell swoop. Something like:

```
mutate(
  facility_type = factor(
    facilityType, 
    labels = c("manufacturing", "office", "research and development", "other")
  )
)
```

Then you might have to drop the old column afterwards. Can you try this?